src/dotnet/commands/dotnet-tool/install/LocalizableStrings.resx

```diff
@@ -161,7 +161,9 @@ Tool '{1}' (version '{2}') was successfully installed.</value>
     <value>The settings file in the tool's NuGet package is invalid: {0}</value>
   </data>
   <data name="ToolInstallationFailed" xml:space="preserve">
-    <value>Tool '{0}' failed to install.</value>
+    <value>Tool '{0}' failed to install. You may need to specify the version using
+
+   dotnet tool install -g t-rex --version &lt;version&gt;</value>
```

Comment:
What is `t-rex`?  Why not `{0}`?  Sorry, I didn't have a change to review this earlier today.

Reply:
Also, since this message is now displayed upon every failure, this should probably be more specific:

`Tool '{0}' failed to install. If you are attempting to install a preview version of the tool, you may need to specify the version using:
dotnet tool install -g {0} --version &lt;version&gt;
`

Reply:
One more thought: this message should be scoped to packages install failures.